app/src/NativeBase/Containers/ProfileContainer.tsx

```diff
@@ -13,20 +53,111 @@ import React from 'react'
 const ProfileContainer = () => {
   return (
     <VStack
-      paddingTop="4"
-      paddingBottom="2"
-      alignItems="center"
-      space="4"
-      backgroundColor="secondaryGrey.60"
+      padding={2}
+      backgroundColor="bg.100"
       _dark={{ backgroundColor: 'bgDarkMode.100' }}
     >
-      <Center
-        backgroundColor="bg.100"
-        _dark={{ backgroundColor: 'bgDarkMode.100' }}
-        width="100%"
-      >
-        <Heading size="sm">My Profile</Heading>
-      </Center>
+      <ScrollView>
+        <VStack alignItems="center" fontSize="xs">
+          <Heading fontFamily="title" size="lg">
+            My Profile
+          </Heading>
+          <HStack marginTop={4}>
+            <Box
+              height="93px"
+              width="93px"
+              borderColor="darkGrey.100"
+              _dark={{ borderColor: 'secondaryGrey.40' }}
+              borderWidth="4px"
+              borderRadius="full"
+              marginLeft="24px"
+            />
+            <Icon
+              as={MaterialIcons}
+              color="darkGrey.100"
+              _dark={{ color: 'secondaryGrey.40' }}
+              name="lock"
```

Comment:
What's the padlock icon for in the designs / why is it needed?  (do we need it?)

Reply:
I have no idea, haha. I did leave that out for that reason, but David quite rightly mentioned it was missing so now I've added it as I reckon it would be easier removed once checked with Mia and Joanna than added.

Reply:
I've asked Mia what it is for so will wait for reply. If it is for future version we could also comment it out before merging. 
Same with the edit button. It will be needed for the profile screen David is going to work on next but it still might be better to comment it out before merging. Thoughts?

Reply:
Mia confirmed the padlock is for the option where the user can choose whether to make the profile public or not. Think you are right and that this will not be part of MVP+1. We can delete or comment out until use. I'm happy either way.

Reply:
Maybe comment out for now?  That way the work you've done can still be used by someone in the future

Reply:
I've removed it as it included an added margin to the profile picture to ensure it remained centred. Might be confusing. Also a real chance this feature will look different if and when it get implemented.